grails-app/utils/org/broadinstitute/orsp/utils/UtilityClass.groovy

```diff
@@ -78,6 +79,26 @@ class UtilityClass {
         }
     }
 
+    /**
+     * Register Issue's JSON mapping to return its transient properties
+    */
+    static void registerIssueForSampleDataCohortsMarshaller() {
```

Comment:
nit: This is mostly an expansion of `registerIssueMarshaller` minus the `reviewCategory` stuff. Why not just add the new things to `registerIssueMarshaller` and use that? Having two different DTOs that are essentially the same thing is a maintenance headache. You'll have two different APIs that essentially do the exact same thing. I strongly recommend simplification and re-use instead of duplication.

Reply:
I agree. I've removed this unnecessary marshaller,